test/suites/drift/suite_test.go

```diff
@@ -269,4 +271,116 @@ var _ = Describe("Drift", Label("AWS"), func() {
 		env.ExpectUpdated(pod)
 		env.EventuallyExpectNotFound(pod, node)
 	})
+	Context("Provisioner Drift", func() {
+		var pod *v1.Pod
+		var provider *v1alpha1.AWSNodeTemplate
+		var testProvisionerOptions test.ProvisionerOptions
+		var provisioner *v1alpha5.Provisioner
+		BeforeEach(func() {
```

Comment:
You can potentially use a `DescribeTable` here to express all the fields; though, I think that Nick is right that decreasing the timing on this might be the best move by launching different go threads to create different provisioners that you can update to reduce the overall test time